build/bootable/root/etc/systemd/system/resizefs.service

```diff
@@ -7,8 +7,6 @@ After=repartition.service
 Type=oneshot
 ExecStart=/etc/vmware/resize_volume.sh resize /dev/sda 2
 ExecStart=/etc/vmware/resize_volume.sh resize /dev/sdb 1
-ExecStart=/etc/vmware/resize_volume.sh resize /dev/sdc 1
-ExecStart=/etc/vmware/resize_volume.sh resize /dev/sdd 1
```

Comment:
You need to update `neutron/build/bootable/root/etc/systemd/system/appliance-mounts.target` as well.

Reply:
And remove the undeeded pci paths from `
        neutron/build/bootable/root/etc/systemd/system/repartition.service
      `.